### PR TITLE
Add South America file

### PR DESCRIPTION
**Pull request by qnadeem1**
add south america file

### CODE REVIEW

File: South America [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Brazil
+Argentina
```
Comment (knadeem11) [inline South America:2]: are the spellings of the paragoye correct?

File: South America [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+Paragoye
```
Comment (knadeem11) [inline South America:3]: starting a review

**Review by knadeem11 — COMMENTED** [review]
looking good.

File: South America (hunk 1 above)
Comment (qnadeem1) [reply South America:2]: yes

File: South America (hunk 2 above)
Comment (qnadeem1) [reply South America:3]: cool

**Pull request merged**